test/controllers/AmendExportedWeightControllerSpec.scala

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import base.SpecBase
+import connectors.CacheConnector
+import forms.AmendExportedWeightFormProvider
+import models.Mode.NormalMode
+import models.UserAnswers
+import navigation.{FakeNavigator, Navigator}
+import org.mockito.ArgumentMatchersSugar.any
+import org.mockito.Mockito.when
+import org.scalatestplus.mockito.MockitoSugar
+import pages.AmendExportedWeightPage
+import play.api.inject.bind
+import play.api.mvc.Call
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import uk.gov.hmrc.http.HttpResponse
+import views.html.AmendExportedWeightView
+
+import scala.concurrent.Future
+
+// todo: fix tests
```

Comment:
remember me

Reply:
change comment to: // todo: replace test with proper real test, that do not start the application